### ISSUE TITLE
Not working with 0.7.5

### ISSUE THREAD

**Issue by mech-tools** [illandril/FoundryVTT-token-hud-scale#5]
Hello,

I'm using 0.7.5 and DnD 0.98.
Activating the module doesn't change status effects on a token.

**Comment by illandril** [comment]
I cannot reproduce this issue. Please double-check to make sure you are using v1.0.2 of Illandril's Token HUD Scaler, and that it is not only installed but active for your world.

If Token HUD Scaler is up-to-date and active, try disabling all other modules.
If the module works when it is the only module enabled, please enable your other modules one at a time until it stops working, and then let me know which other module(s) you have enabled when it stops working.

If the module still does not work when it is the only one enabled, try using your browser in Incognito mode, with all browser extensions disabled.
If the module still does not work in Incognito mode...
1. Let me know which browser (including the specific browser version) you are using
2. Open the DevTools (F12) and let me know if there are any errors in the console

If the module works in Incognito mode...
Go back to non-Incognito mode and clear your browser's cache and trying again.

If the module works after clearing your cache, then it must've been a caching issue where you had and older version of some file(s) loaded - let me know and close this issue.

If the module still doesn't work after clearing you cache...
1. Let me know which browser (including the specific browser version) you are using
2. Open the DevTools (F12) and let me know if there are any errors in the console
3. Let me know what browser extensions you have installed

**Comment by mech-tools** [comment]
![image](https://user-images.githubusercontent.com/1687854/97021062-a2eee980-1552-11eb-98b9-fb46cc197ebe.png)

All modules deactivated but yours.
Incognito mode as well.

Win 10
Chrome 86.0.4240.111
No errors in the console either at init or when applying a status effect

**Comment by mech-tools** [comment]
Same in a test world without any data initialized by other modules.

Blank world. Created an empty actor. Placed the token on an empty map.
Same display.

**Comment by illandril** [comment]
Thanks for the info - I've reproduced it now.

It turns out that newly created status effects are stored differently from previously created status effects. Previously created status effects work properly still, so everything looked good to me when I tested it, but newly created status effects are broken.

This looks like it will be a simple fix, so I should be able to squeeze it in during my lunch break and have a fix ready today.
